sklearn/decomposition/sparse_pca.py

```diff
@@ -77,6 +77,12 @@ class SparsePCA(BaseEstimator, TransformerMixin):
     n_iter_ : int
         Number of iterations run.
 
+    variance : bool, optional
+        If true, compute the explained variance
+
```

Comment:
sorry for the miss leading. variance should be a parameter (I tend to use the word attribute for both as in C++...)